.gitmodules

```
@@ -26,3 +26,6 @@
 	path = extensions/FlarumAuth
 	url = https://github.com/archlinux-de/mediawiki-extensions-FlarumAuth.git
 	branch = REL1_38
+[submodule "extensions/mediawiki-extensions-DarkMode"]
+	path = extensions/mediawiki-extensions-DarkMode
+	url = https://github.com/wikimedia/mediawiki-extensions-DarkMode
```

The git submodule branch needs to match the used mediawiki release.

```suggestion
[submodule "extensions/DarkMode"]
	path = extensions/DarkMode
	url = https://github.com/wikimedia/mediawiki-extensions-DarkMode.git
	branch = REL1_38
```


The submodule currently should be at commit `fe470966d18fa4749f9b92097052fcd031e3cad6`.